libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/UIDevice+SFHardware.m

```diff
@@ -17,72 +17,117 @@
 @implementation UIDevice (SFHardware)
 /*
  Platforms
+
+ iFPGA ->        Unknown or Internal Development Platform
  
- iFPGA ->        ??
- 
- iPhone1,1 ->    iPhone 1G, M68
- iPhone1,2 ->    iPhone 3G, N82
- 
- iPhone2,1 ->    iPhone 3GS, N89/N88?
- 
- iPhone3,1 ->    iPhone 4/AT&T, N90
- iPhone3,2 ->    iPhone 4/Other Carrier?, ??
- iPhone3,3 ->    iPhone 4/Verizon, N92
- 
- iPhone4,1 ->    iPhone 4S/GSM, N94
- iPhone4,2 ->    iPhone 4S/???, ???
- iPhone4,3 ->    iPhone 4S/???, ???
+ // iPhone Models
```

Comment:
Can we take out the ones we don't support in the commented list too?

Reply:
Yes. I'll update that